res/css/views/messages/_DecryptionFailureBody.pcss

```diff
@@ -10,3 +10,24 @@ Please see LICENSE files in the repository root for full details.
     color: $secondary-content;
     font-style: italic;
 }
+
+/* Formatting for the "Verified identity has changed" error */
+.mx_DecryptionFailureVerifiedIdentityChanged > span {
+    /* Show it in red */
+    color: $e2e-warning-color;
```

Comment:
Can we use instead directly a compound token: `Light/color/border/critical/subtle`